worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/QueryTool.java

```diff
@@ -53,9 +52,8 @@ public boolean canUse(Actor player) {
     public boolean actPrimary(Platform server, LocalConfiguration config, Player player, LocalSession session, Location clicked, @Nullable Direction face) {
 
         World world = (World) clicked.getExtent();
-        EditSession editSession = session.createEditSession(player);
         BlockVector3 blockPoint = clicked.toVector().toBlockPoint();
-        BaseBlock block = editSession.getFullBlock(blockPoint);
+        BaseBlock block = player.getExtent().getFullBlock(blockPoint);
```

Comment:
```suggestion
        BaseBlock block = clicked.getExtent().getFullBlock(blockPoint);
```
unlikely a player will be able to click a block in some extent they aren't in, but just to be pedantic.